### PR TITLE
Translate glossaries tagged as fundamental into Bahasa

### PR DESCRIPTION
**Pull request by habibrosyad**
This PR translates all terms tagged as `fundamental` in https://k8s.io/docs/reference/glossary/?fundamental=true

/cc @ariscahyadi @girikuncoro

### CODE REVIEW

**Comment by habibrosyad** [comment]
Preview https://deploy-preview-24347--kubernetes-io-master-staging.netlify.app/id/docs/reference/glossary/?fundamental=true

File: content/id/docs/reference/glossary/minikube.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+aka:
+tags:
+- fundamental
+- tool
+---
+Sebuah utilitas untuk menjalankan Kubernetes secara lokal.
+
+<!--more-->
+
+Minikube menjalankan klaster dengan satu Node di dalam sebuah VM pada komputermu. Kamu dapat menggunakan Minikube untuk [mencoba Kubernetes dalam lingkungan pembelajaran](/docs/setup/learning-environment/).
```
Comment (ariscahyadi) [inline content/id/docs/reference/glossary/minikube.md:18]: I think ID docs is available now :)

File: content/id/docs/reference/glossary/pod-lifecycle.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: Siklus Hidup Pod
+id: pod-lifecycle
+date: 2019-02-17
+full-link: /docs/concepts/workloads/pods/pod-lifecycle/
```
Comment (ariscahyadi) [inline content/id/docs/reference/glossary/pod-lifecycle.md:5]: Link ID is available!

File: content/id/docs/reference/glossary/pod.md [hunk 3]
```diff
@@ -2,16 +2,16 @@
...
 tags:
 - core-object
 - fundamental
 ---
-Unit Kubernetes yang paling sederhana dan kecil. Sebuah Pod merepresentasikan sebuah set kontainer yang dijalankan {{< glossary_tooltip text="kontainer" term_id="container" >}} pada kluster kamu.
+Unit Kubernetes yang paling sederhana dan kecil. Sebuah Pod merepresentasikan sebuah set {{< glossary_tooltip term_id="container" >}} yang dijalankan pada klustermu.
 
 <!--more--> 
-Sebuah Pod biasanya digunakan untuk menjalankan sebuah kontainer. Pod juga dapat digunakan untuk menjalankan beberapa sidecar container dan beberapa fiture tambahan. Pod biasanya diatur oleh sebuah {{< glossary_tooltip term_id="deployment" >}}.
+Sebuah Pod biasanya digunakan untuk menjalankan sebuah Container. Pod juga bisa menjalankan _sidecar_ Container yang akan menambahkan fitur ekstra seperti _logging_. Pod biasanya dikelola oleh sebuah {{< glossary_tooltip term_id="deployment" >}}.
```
Comment (ariscahyadi) [inline content/id/docs/reference/glossary/pod.md:17]: "logging" = "pencatatan"

File: content/id/docs/reference/glossary/quantity.md [hunk 4]
```diff
@@ -1,22 +1,20 @@
...
 
-<!--lebih lanjut-->
+<!--more-->
 
-Kuantitas adalah representasi dari bilangan kecil atau besar menggunakan notasi 
-bilangan bulat kompak dengan sufiks SI. Bilangan pecahan direpresentasikan dengan satuan mili, 
-sedangkan bilangan besar direpresentasikan dengan satuan kilo, mega, atau giga.
+Kuantitas adalah representasi dari bilangan kecil atau besar menggunakan notasi bilangan bulat kompak dengan sufiks SI. Bilangan pecahan direpresentasikan dengan satuan mili, sedangkan bilangan besar direpresentasikan dengan satuan kilo, mega, atau giga.
 
 Misalnya, angka `1,5` direpresentasikan sebagai` 1500m`, sedangkan angka `1000` dapat direpresentasikan sebagai` 1k`, dan `1000000` sebagai` 1M`.  Kamu juga dapat menentukan sufiks notasi biner; angka 2048 dapat
```
Comment (ariscahyadi) [inline content/id/docs/reference/glossary/quantity.md:19]: nit!
```suggestion
Misalnya, angka `1,5` direpresentasikan sebagai `1500m`, sedangkan angka `1000` dapat direpresentasikan sebagai `1k`, dan `1000000` sebagai `1M`.  Kamu juga dapat menentukan sufiks notasi biner; angka 2048 dapat 
```

File: content/id/docs/reference/glossary/selector.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+title: Selector
+id: selector
+date: 2018-04-12
+full_link: /id/docs/concepts/overview/working-with-objects/labels/
+short_description: >
+  Memungkinkan pengguna untuk memfilter daftar sumber daya berdasarkan label yang digunakan.
```
Comment (ariscahyadi) [inline content/id/docs/reference/glossary/selector.md:7]: "memfilter" = "menyaring"? 
```suggestion
  Memungkinkan pengguna untuk menyaring daftar sumber daya berdasarkan label yang digunakan.
```

File: content/id/docs/reference/glossary/selector.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- fundamental
+---
+Memungkinkan pengguna untuk memfilter daftar sumber daya berdasarkan label yang digunakan.
```
Comment (ariscahyadi) [inline content/id/docs/reference/glossary/selector.md:13]: "memfilter" = "menyaring"?
```suggestion
Memungkinkan pengguna untuk menyaring daftar sumber daya berdasarkan label yang digunakan.
```

File: content/id/docs/reference/glossary/selector.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+<!--more--> 
+
+_Selector_ diterapkan ketika melakukan kueri terhadap daftar sumber daya untuk memfilter mereka berdasarkan labelnya.
```
Comment (ariscahyadi) [inline content/id/docs/reference/glossary/selector.md:17]: "memfilter" = "menyaring"?
```suggestion
_Selector_ diterapkan ketika melakukan kueri terhadap daftar sumber daya untuk menyaring mereka berdasarkan labelnya.
```

File: content/id/docs/reference/glossary/service.md [hunk 8]
```diff
@@ -2,22 +2,22 @@
 title: Service
 id: service
 date: 2020-04-05
-full_link: /docs/concepts/services-networking/service/
+full_link: /id/docs/concepts/services-networking/service/
 short_description: >
-  Sebuah Cara untuk mengekspos aplikasi yang berjalan pada sebuah kumpulan Pod sebagai layanan jaringan.
+  Cara untuk mengekspos aplikasi yang berjalan pada satu set Pod sebagai layanan jaringan.
```
Comment (ariscahyadi) [inline content/id/docs/reference/glossary/service.md:7]: "set" = "kumpulan"
```suggestion
  Cara untuk mengekspos aplikasi yang berjalan pada satu kumpulan Pod sebagai layanan jaringan.
```

File: content/id/docs/reference/glossary/statefulset.md [hunk 9]
```diff
@@ -2,22 +2,21 @@
...
-aka: 
+aka:
 tags:
 - fundamental
 - core-object
 - workload
 - storage
 ---
- Melakukan proses manajemen deployment dan _scaling_ dari sebuah set {{< glossary_tooltip text="Pods" term_id="pod" >}}, *serta menjamin mekanisme _ordering_ dan keunikan* dari Pod ini.
+Mengelola penggelaran dan penyekalaan dari suatu set {{< glossary_tooltip term_id="pod" >}}, *serta menjamin  urutan dan keunikan* dari masing-masing Pod dalam set tersebut.
```
Comment (ariscahyadi) [inline content/id/docs/reference/glossary/statefulset.md:16]: "set" = "kumpulan"?

File: content/id/docs/reference/glossary/statefulset.md [hunk 10]
```diff
@@ -2,22 +2,21 @@
...
 
-<!--more--> 
+<!--more-->
 
-Seperti halnya {{< glossary_tooltip term_id="deployment" >}}, sebuah StatefulSet akan melakukan proses manajemen Pod yang didasarkan pada spec container identik. Meskipun begitu tidak seperti sebuah Deployment, sebuah StatefulSet akan menjamin identitas setiap Pod yang ada. Pod ini akan dibuat berdasarkan spec yang sama, tetapi tidak dapat digantikan satu sama lainnya&#58; setiap Pod memiliki identifier persisten yang akan di-maintain meskipun pod tersebut di (re)schedule.
-
-Sebuah StatefulSet beroperasi dengan pola yang sama dengan Kontroler lainnya. Kamu dapat mendefinisikan state yang diinginkan pada objek StatefulSet, dan kontroler StatefulSet akan membuat update yang dibutuhkan dari _state_ saat ini.
+Seperti halnya {{< glossary_tooltip term_id="deployment" >}}, StatefulSet akan mengelola Pod dengan spesifikasi Container yang identik. Tidak seperti Deployment, StatefulSet menjamin pengenal yang tetap (_sticky_) untuk setiap Pod. Kesemua Pod tersebut dibuat berdasarkan spesifikasi yang sama, akan tetapi tidak dapat digantikan satu sama lain: masing-masing Pod memiliki pengenal tetap yang akan dipertahankan setiap penjadwalan ulang terjadi.
```
Comment (ariscahyadi) [inline content/id/docs/reference/glossary/statefulset.md:20]: nit!
```suggestion
Seperti halnya {{< glossary_tooltip term_id="deployment" >}}, StatefulSet akan mengelola Pod dengan spesifikasi Container yang identik. Tidak seperti Deployment, StatefulSet menjamin pengenal yang tetap (_sticky_) untuk setiap Pod. Semua Pod tersebut dibuat berdasarkan spesifikasi yang sama, akan tetapi tidak dapat digantikan satu sama lain: masing-masing Pod memiliki pengenal tetap yang akan dipertahankan setiap penjadwalan ulang terjadi.
```

File: content/id/docs/reference/glossary/workload.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+tags:
+- fundamental
+---
+Sebuah beban kerja merupakan suatu aplikasi yang berjalan pada Kubernetes.
+
+<!--more-->
+
+Beragam objek inti yang merepresentasikan tipe atau bagian berbeda dari suatu beban kerja termasuk objek DaemonSet, Deployment, Job, ReplicaSet, dan StatefulSet.
+
+Sebagai contoh, suatu beban kerja dengan sebuah server web dan pangkalan data mungkin menjalankan pangkalan data dalam satu {{< glossary_tooltip term_id="StatefulSet" >}} dan server web dalam satu {{< glossary_tooltip term_id="Deployment" >}}.
```
Comment (ariscahyadi) [inline content/id/docs/reference/glossary/workload.md:20]: nit!
```suggestion
Sebagai contoh, suatu beban kerja dengan sebuah server jejaring (_web_) dan pangkalan data mungkin menjalankan pangkalan data dalam satu {{< glossary_tooltip term_id="StatefulSet" >}} dan server web dalam satu {{< glossary_tooltip term_id="Deployment" >}}.
```

File: content/id/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md [hunk 12]
```diff
@@ -426,7 +426,7 @@ pengambilan metrik. Terakhir, kondisi terakhir, `ScalingLimited`, menunjukkan ba
 
 ## Lampiran: Kuantitas
 
-Semua metrik di HorizontalPodAutoscaler dan metrik API ditentukan menggunakan notasi bilangan bulat khusus yang dikenal di Kubernetes sebagai {{< glossary_tooltip term_id="kuantitas" text="kuantitas">}}. Misalnya, kuantitas `10500m` akan ditulis sebagai `10.5` dalam notasi desimal. Metrik API akan menampilkan bilangan bulat tanpa sufiks jika memungkinkan, dan secara umum akan mengembalikan kuantitas dalam satuan mili.  Ini berarti Anda mungkin melihat nilai metrik Anda berfluktuasi antara `1` dan` 1500m`, atau `1` dan` 1,5` ketika ditulis dalam notasi desimal.
+Semua metrik di HorizontalPodAutoscaler dan metrik API ditentukan menggunakan notasi bilangan bulat khusus yang dikenal di Kubernetes sebagai {{< glossary_tooltip term_id="quantity" text="kuantitas">}}. Misalnya, kuantitas `10500m` akan ditulis sebagai `10.5` dalam notasi desimal. Metrik API akan menampilkan bilangan bulat tanpa sufiks jika memungkinkan, dan secara umum akan mengembalikan kuantitas dalam satuan mili.  Ini berarti Anda mungkin melihat nilai metrik Anda berfluktuasi antara `1` dan` 1500m`, atau `1` dan` 1,5` ketika ditulis dalam notasi desimal.
```
Comment (ariscahyadi) [inline content/id/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md:429]: Use "kamu" instead "anda"!
```suggestion
Semua metrik di HorizontalPodAutoscaler dan metrik API ditentukan menggunakan notasi bilangan bulat khusus yang dikenal di Kubernetes sebagai {{< glossary_tooltip term_id="quantity" text="kuantitas">}}. Misalnya, kuantitas `10500m` akan ditulis sebagai `10.5` dalam notasi desimal. Metrik API akan menampilkan bilangan bulat tanpa sufiks jika memungkinkan, dan secara umum akan mengembalikan kuantitas dalam satuan mili.  Ini berarti kamu mungkin melihat nilai metrik kamu berfluktuasi antara `1` dan` 1500m`, atau `1` dan `1,5` ketika ditulis dalam notasi desimal.
```

**Review by ariscahyadi — COMMENTED** [review]
Thanks @habibrosyad for your contribution! I have several comments to be addressed.

File: content/id/docs/reference/glossary/docker.md [hunk 13]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- fundamental
+---
+Docker (secara spesifik, Docker Engine) merupakan suatu teknologi perangkat lunak yang menyediakan virtualisasi pada level sistem operasi yang juga dikenal sebagai {{< glossary_tooltip term_id="container" >}}.
+
+<!--more-->
+
+Docker menggunakan fitur isolasi sumber daya pada kernel Linux seperti cgroup dan _namespace_, dan [UnionFS](https://docs.docker.com/get-started/overview/#union-file-systems) seperti OverlayFS dan lainnya untuk memungkinkan masing-masing Container dijalankan pada satu instans Linux, menghindari beban tambahan (_overhead_) saat memulai dan menjalankan VM.
```
[the remark this answers is not in the document]
Comment (habibrosyad) [reply content/id/docs/reference/glossary/docker.md:17]: Based on the context that's a Linux kernel feature (not particularly a Kubernetes' `Namespace`, should that also be written as `Namespace`?

File: content/id/docs/reference/glossary/service.md (hunk 8 above)
Comment (irvifa) [reply content/id/docs/reference/glossary/service.md:7]: For set just make sure we’re consistent on whether using set or kumpulan here

File: content/id/docs/reference/glossary/kube-apiserver.md [hunk 14]
```diff
@@ -4,16 +4,15 @@ id: kube-apiserver
...
 - fundamental
 ---
- Komponen di master yang mengekspos API Kubernetes. Merupakan <i> front-end </i> dari <i> kontrol plane </i> Kubernetes.
+Komponen _control plane_ yang mengekspos API Kubernetes. Merupakan _front-end_ dari _control plane_ Kubernetes.
 
 <!--more-->
 
-Komponen ini didesain agar dapat di-<i>scale</i> secara horizontal. Lihat [Membangun Klaster HA](/docs/admin/high-availability/).
-
+Komponen ini didesain agar dapat diskalakan secara horizontal. Lihat [Membangun Klaster HA](/docs/admin/high-availability/).
```
[the remark this answers is not in the document]
Comment (habibrosyad) [reply content/id/docs/reference/glossary/kube-apiserver.md:18]: According to https://kbbi.kemdikbud.go.id/entri/horizontal `horizontal` is the standard..

File: content/id/docs/reference/glossary/selector.md (hunk 5 above)
Comment (habibrosyad) [reply content/id/docs/reference/glossary/selector.md:7]: memfilter is a valid kbbi https://kbbi.web.id/filter, and has been used in other localisations too, e.g. https://support.microsoft.com/id-id/office/memfilter-data-dalam-rentang-atau-tabel-01832226-31b5-4568-8806-38c37dcc180e

File: content/id/docs/reference/glossary/selector.md (hunk 6 above)
Comment (habibrosyad) [reply content/id/docs/reference/glossary/selector.md:13]: memfilter is a valid kbbi https://kbbi.web.id/filter, and has been used in other localisations too, e.g. https://support.microsoft.com/id-id/office/memfilter-data-dalam-rentang-atau-tabel-01832226-31b5-4568-8806-38c37dcc180e

File: content/id/docs/reference/glossary/selector.md (hunk 7 above)
Comment (habibrosyad) [reply content/id/docs/reference/glossary/selector.md:17]: memfilter is a valid kbbi https://kbbi.web.id/filter, and has been used in other localisations too, e.g. https://support.microsoft.com/id-id/office/memfilter-data-dalam-rentang-atau-tabel-01832226-31b5-4568-8806-38c37dcc180e

File: content/id/docs/reference/glossary/workload.md (hunk 11 above)
Comment (habibrosyad) [reply content/id/docs/reference/glossary/workload.md:20]: web is valid kbbi too https://kbbi.web.id/web, and has been used, e.g. here https://id.wikipedia.org/wiki/Server_web

File: content/id/docs/reference/glossary/logging.md [hunk 15]
```diff
@@ -0,0 +1,18 @@
+---
+title: Logging
```
[the remark this answers is not in the document]
Comment (habibrosyad) [reply content/id/docs/reference/glossary/logging.md]: I put _Logging_ as aka.

**Comment by habibrosyad** [comment]
Most of the comments have been addressed, some are arguable. Could you check again @ariscahyadi?

**Comment by irvifa** [comment]
This PR is massive. I suggest next time separate it to smaller PR for example one word one PR and I believe even for that the changes is still massive.

**Comment by sftim** [comment]
@habibrosyad would you be willing to split this into several smaller PRs?

**Comment by habibrosyad** [comment]
@irvifa @sftim Sure.

If you noticed my previous PRs for Indonesian glossary, I try to split them into a PR per term. Got a suggestion on my last PR to combine several terms into one, and I'm thinking of doing it per tag. Since we usually translate a page in the size of hundreds lines, so I was thinking that this is still acceptable.

Thanks for the suggestions.

**Comment by sftim** [comment]
I can't speak specifically for the bahasa Indonesia localization team, but in both work for Kubernetes and my day job in IT consultancy, I actively favor PRs that add the smallest piece of incremental value, _especially_ if those changes won't conflict with each other when merged.

**Comment by habibrosyad** [comment]
> I actively favor PRs that add the smallest piece of incremental value, especially if those changes won't conflict with each other when merged.

Me to personally. 

There are different preferences for the bahasa localisation reviewers which we haven't actually discussed in depth yet.

**Comment by ariscahyadi** [comment]
Hi @habibrosyad @irvifa @sftim,

I expect group of glossary per PR, not a single glossary per PR... but I didn't expect that it will be 60+ at once :)

**Comment by habibrosyad** [comment]
@ariscahyadi haha, my mistake here, because I treat it just like translating a full page (all terms tagged with `fundamental` = appear as 1 page).

@irvifa @ariscahyadi @girikuncoro wdyt, should I close this PR and divide it into smaller ones? if so, then how 'small' it will be?

**Comment by girikuncoro** [comment]
Yes let's make smaller PRs, you can group few glossaries per PR, I'm fine
with that 👍

**Comment by habibrosyad** [comment]
Sure, so I'm going to

/close

this one.

**Comment by ariscahyadi** [comment]
> @ariscahyadi haha, my mistake here, because I treat it just like translating a full page (all terms tagged with `fundamental` = appear as 1 page).
> 
> @irvifa @ariscahyadi @girikuncoro wdyt, should I close this PR and divide it into smaller ones? if so, then how 'small' it will be?
@habibrosyad I think 5-10 glossaries will be good!

**Comment by habibrosyad** [comment]
@ariscahyadi the 3 new PRs that I just made contains 2 actually, since @irvifa seems to be eager with 1. 

OK, will do 5 for the upcoming PRs.

**Pull request closed without merging**